docs/ko/msg_docs/BatteryStatus.md

```diff
@@ -25,7 +25,6 @@ uint8 priority					# Zero based priority is the connection on the Power Controll
 uint16 capacity					# actual capacity of the battery
 uint16 cycle_count				# number of discharge cycles the battery has experienced
 uint16 average_time_to_empty			# predicted remaining battery capacity based on the average rate of discharge in min
-uint16 serial_number				# serial number of the battery pack
```

Comment:
All changes to the docs/tree should be dealt with separately. The tooling is supposed to auto-update these from the sources. In particular, these language specific pages would be best undone (i.e. docs/uk, docs/zh etc) because this makes it harder to round trip with crowdin

Reply:
I'm aware it should be done automatically but currently isn't so I thought I would just do manually until then but can revert if it generates more churn.

Reply:
I did that in place: e584afa954facb95a649b5daede2e980fb62ecfc